host/install.sh

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env bash
```

Comment:
set execute permission as default for install.sh

Reply:
done updated that now, thank you